components/Survey/Survey.js

```diff
@@ -56,20 +58,20 @@ export default class Survey extends React.Component {
       binarySearchMid: mid
     });
     if (lo >= hi) {
-      const surveyResults = { intensity: this.questions[mid].intensity }
-      this.setState({ 
+      const surveyResults = { intensity: this.questions[mid].intensity };
+      this.setState({
         surveyResults: surveyResults
       });
-      this.onSurveyCompleted(surveyResults);
+      this.onSurveyCompleted( surveyResults.intensity );
```

Comment:
Aquí lo que habíamos conversado el otro día

Debería ser `this.onSurveyCompleted(surveyResults)`